### PR TITLE
Change style of chat bubble#2174, Integrate Styled Components in ChatApp #2175

### LINKED ISSUE #2174: Change style of chat bubble
**Actual Behaviour**

Old style present

**Expected Behaviour**

Change to a newer style, add chat bubble arrow, add styled components to chat component

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2174, #2175 

Changes:

- Change style of the chat bubble

- Integrate Styled Components in ChatApp and its child components

- CSS Cleanup

Demo Link: https://pr-2176-fossasia-susi-web-chat.surge.sh

Screenshots of the change: 
Screenshot 2019-06-05 at 5 00 00 AM

### CODE REVIEW

File: src/components/ChatApp/MessageListItem/helperFunctions.react.js [hunk 1]
```diff
@@ -20,17 +20,28 @@ import CardActionArea from '@material-ui/core/CardActionArea';
...
 import Typography from '@material-ui/core/Typography';
 import { injectIntl } from 'react-intl';
+import styled from 'styled-components';
 
 const styles = {
-  footerStyle: {
-    display: 'block',
-    float: 'left',
-  },
   indicatorStyle: {
```
Comment (harsh-jindal) [inline src/components/ChatApp/MessageListItem/helperFunctions.react.js]: Change this into styled component

File: src/components/ChatApp/MessageListItem/helperFunctions.react.js [hunk 2]
```diff
@@ -189,7 +198,6 @@ function urlDomain(data) {
 // Draw Tiles for Websearch RSS data
 export function drawCards(tilesData) {
   const titleStyle = {
```
Comment (harsh-jindal) [inline src/components/ChatApp/MessageListItem/helperFunctions.react.js]: Change this to styled component

**Comment by harsh-jindal** [comment]
@fragm3 Also fix this
Screenshot 2019-06-05 at 8 45 03 AM

**Comment by fragm3** [comment]
@harsh-jindal The link styling got changed due to the upgrade of material-ui. I have fixed it in https://github.com/fossasia/susi.ai/pull/2172. Won't be a problem anymore.

**Comment by fragm3** [comment]
@harsh-jindal made the changes.

**Pull request merged**